context/app/static/js/shared-styles/scrollShadows.js

```diff
@@ -0,0 +1,48 @@
+import { css } from 'styled-components';
+import { isMobileSafari } from 'react-device-detect';
+
+// See https://css-tricks.com/scroll-shadows-with-javascript/ for reference.
+// The scroll shadows will not work on iOS Safari.
```

Comment:
Maybe add a comment explaining why supporting Safari wasn't workable? (Imagine someone in the future is tasked with adding support: What would you tell them?)